tasks/uwsgi.yml

```diff
@@ -0,0 +1,11 @@
+# set directory permissions for uswgi for logs and pid file
+- file:
+  path: {{ openwisp2_path }}
+  owner:  www-data
+  mode: 0744
+
+- file:
+  path: {{ openwisp2_path }}/log
+  owner:  www-data
+  mode: 0744
+  
```

Comment:
add a new line at the end of the file (configure your editor to do this automatically)

Reply:
okay, also need to make some changes and remove the second path